keps/sig-security/3203-auto-refreshing-official-cve-feed/README.md

```diff
@@ -0,0 +1,475 @@
+-->
+
+Items marked with (R) are required *prior to targeting to a milestone / release*
+.
+
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir
+  in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and
+  SIG Testing input (including test refactors)
+  - [ ] e2e Tests for all Beta API Operations (endpoints)
+  - [ ] (R) Ensure GA e2e tests for meet requirements
+    for [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md)
+  - [ ] (R) Minimum Two Week Window for GA e2e tests to prove flake free
+- [ ] (R) Graduation criteria is in place
+  - [ ] (
+    R) [all GA Endpoints](https://github.com/kubernetes/community/pull/1806)
+    must be hit
+    by [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md)
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for
+  publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to
+  mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+Currently it is not possible to filter for issues or PRs that are related to
+CVEs announced by kubernetes. This KEP addresses this concern by labelling this
+issues or PRs with the new label **official-cve-feed** using the automation. The
+in-scope issues are the closed issues for which there is a CVE ID and is
+officially announced as a Kubernetes CVE by SRC in the past.
+
+## Motivation
+
+With the growing number of eyes on Kubernetes, the number of CVEs related to
+Kubernetes have increased. Although most CVEs are regularly fixed that directly
+or indirectly or transitively impact Kubernetes, there is no single place to
+programmatically subscribe or pull the data of fixed CVEs, for the end users of
+Kubernetes. Current options are either broken or incomplete.
+
+An auto-refreshing CVE feed will allow end users to programmatically fetch the
+list of CVEs and allow them to get the latest information from kubernetes
+community.
+
+### Goals
+
+Create a periodically auto-refreshing list of official Kubernetes CVEs
```

Comment:
I'd make machine-readability an explicit goal. To some extent https://groups.google.com/g/kubernetes-security-announce already covers this in a non-machine readable form.

Reply:
IIUC, you mean the feed should be parseable in a well-recognized semi-structured format like JSON but not limited to JSON, correct?

Reply:
@tallclair okay to resolve the comment assuming my understanding above is correct? We are already publishing the blob in JSON

Reply:
Made this explicit in Goals section